domains/kaju.json

```diff
@@ -0,0 +1,11 @@
+{
+	"owner": {
+		"username": "tanishq-singh-2407",
+		"email": "hi@tanishqsingh.com"
+	},
+	"record": {
+		"MX": ["mx.mailtie.com"],
+		"TXT": "mailtie={https://denomail.deno.dev}",
+        "URL": "https://tempmail.tanishqsingh.com/"
```

Comment:
Would like you to add more stuff to your website here (or you could just only have MX and TXT records if you just need email)

Reply:
I just need email

Reply:
> I just need email

Then remove the URL line (also don't forget about the comma after TXT line)